NOTEA/Controllers/UserController.cs

```diff
@@ -59,7 +64,8 @@ public IActionResult LogIn()
         [HttpPost]
         public IActionResult LogIn(string username, string password)
         {
-            UserModel user = new UserModel(username, password);
+            UserModel user = new UserModel(username, password, "");
+            //PAVOJUS KAS CIA REIKIA EMAILO ALIO NORIU KAD NEREIKETU
```

Comment:
Čia reiks perdaryti šitai

Reply:
taip pritariu

Reply:
ten dar kazkur yra panasi vieta bet i'll fix it xd